CONTRIBUTING.md

```diff
@@ -2,26 +2,88 @@
+* [autocomplete-plus](https://github.com/atom/autocomplete-plus) - autocompletions shown while typing. Some languages have additional packages for autocompletion functionality, such as [autocomplete-html](https://github.com/atom/autocomplete-html).
+* [git-diff](https://github.com/atom/git-diff) - Git change indicators shown in the editor's gutter.
+* [language-javascript](https://github.com/atom/language-javascript) - all bundled languages are packages too, and each one has a separate package `language-[name]`. Use these for feedback on syntax highlighting issues that only appear for a specific language.
+* [one-dark-ui](https://github.com/atom/one-dark-ui) - the default UI styling for anything but the text editor. UI theme packages (i.e. packages with a `-ui` suffix) provide only styling and it's possible that a bundled package is responsible for a UI issue. There are other other bundled UI themes, such as [one-light-ui](https://github.com/atom/one-light-ui).
+* [one-dark-syntax](https://github.com/atom/one-dark-syntax) - the default syntax highlighting styles applied for all languages. There are other other bundled syntax themes, such as [solarized-dark](https://github.com/atom/solarized-dark). You should use these packages for reporting issues that appear in many languages, but disappear if you change to another syntax theme.
+* [apm](https://github.com/atom/apm) - the `apm` command line tool (Atom Package Manager). You should use this repository for any contributions related to the `apm` tool and to publishing packages.
+* [atom.io](https://github.com/atom/atom.io) - the repository for feedback on the [Atom.io website](https://atom.io) and the [Atom.io package API](https://github.com/atom/atom/blob/master/docs/apm-rest-api.md) used by [apm](https://github.com/atom/apm).
+
+There are many more, but this list should be a good starting point.
+For more information on how to work with Atom's official packages, see [Contributing to Atom Packages](https://github.com/atom/atom/blob/master/docs/contributing-to-packages.md).
 
-* [Submitting Issues](#submitting-issues)
-* [Your First Contribution](#your-first-contribution)
-* [Pull Requests](#pull-requests)
-* [Git Commit Messages](#git-commit-messages)
-* [CoffeeScript Styleguide](#coffeescript-styleguide)
-* [Specs Styleguide](#specs-styleguide)
-* [Documentation Styleguide](#documentation-styleguide)
-* [Issue and Pull Request Labels](#issue-and-pull-request-labels)
+Also, because Atom is so extensible, it's possible that a feature you've become accustomed to in Atom or an issue you're encountering aren't coming from a bundled package at all, but rather a [community package](https://atom.io/packages) you've installed.
+Each community package has its own repository too, and you should be able to find it in Settings > Packages for the packages you installed and contribute there.
 
-## Submitting Issues
+## How can I contribute?
+
+### Your First Contribution
```

Comment:
I feel like "Submitting Issues" should come before "Your First Contribution" because "Your First Contribution" and "Pull Requests" should go together. Also, I feel like "Your First Code Contribution" might be a better name since there are other ways to contribute than submitting code.